spec/fixtures/api/gpu-info.js

```diff
@@ -0,0 +1,9 @@
+const { app } = require('electron')
+
+app.commandLine.appendSwitch('--disable-software-rasterizer')
+
+app.on('ready', async () => {
+  const gpuInfo = await app.getGPUInfo(process.argv.pop())
```

Comment:
When `app.getGPUInfo()` get rejected an exception will be thrown and `console.log()` will never be called.